### ISSUE TITLE
Consider unblocking google-analytics.com

### ISSUE THREAD

**Issue by Yuki2718** [AdguardTeam/AdGuardSDNSFilter#1335]
### Prerequisites

- [X] I checked the documentation and found no answer;
- [X] I checked to make sure that this issue has not already been filed;
- [X] This is not an ad/bug report.

### Problem description

@ameshkov commented in https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1142#issuecomment-1346610510
> Google Analytics and Qualtrics definitely won't be unblocked.

I guess I have to challenge this at least once as it's simply too problematic. For example, https://github.com/AdguardTeam/AdguardFilters/pull/90357 alone, not resolved in DNS, has been [wasting](https://twitter.com/zumusata366/status/1508051498772889608) a [lot](https://twitter.com/search?q=adguard%20paypay&src=typed_query&f=top) of precious time of user, [losing](https://twitter.com/TK_NiMeC3/status/1419997642952609793) financial opportunity, and [causing](https://twitter.com/koumei_wana/status/1576796190183604224) you to lose new user<sup>1</sup>. There are more - this repo has around 10+ GA-related issues and TPF has 200+ exception, and today I found [this](https://twitter.com/otr_da_wa_3/status/1655738351314690050) can be solved if GA is unblocked on DNS without allowing in contents filtering. Note this is not only a matter of DNS user but also of those using DNS filtering on apps. As discussed in Slack in past, there will be many user who only want to block ads and not tracker, but those who want to block GA can still do with personal DNS or with any of client-side AG products. Privacy advocates will critisize you and me if it's done, but the qustion is the total utility of all user.
___
1. Of note someone already [asked](https://twitter.com/miniamaruDEBU/status/1372024462149779456) paypay to solve the issue.

### Proposed solution

Unblock GA in DNS

**Comment by Alex-302** [comment]
This kind of unlocking would make the DNS useless. User can always switch to a non-filtering server or to our free private DNS.

**Comment by jslawler** [comment]
I suppose the same could be said about Google Ads which now breaks [Google's partner ad settings interface](https://github.com/AdguardTeam/AdguardFilters/issues/150327).

Blocking these types of domains which breaks smaller services that aren't used as much I can understand, but when you have large scale applications being broken it may be time to look at improving the approach such as possibly splitting the AdGuard DNS filter into many smaller lists for more granular control.

**Comment by ameshkov** [comment]
Can we consider a different approach and report those breakages to website owners?

I highly doubt that breakage is the intended behaviour and I suppose that once they learn about the issue, they'll try to resolve it.

To do that, we need to know which websites & apps are affected.

**Comment by ameshkov** [comment]
Let's try again, but with more details about why the breakage actually occurs. 

We'll report it from our official Japanese account.

**Comment by ameshkov** [comment]
Let's wait and see if they fix it in the coming month: https://twitter.com/PayPaysupport/status/1658385834062774272

**Comment by Yuki2718** [comment]
BTW https://github.com/AdguardTeam/AdguardFilters/pull/90357 was about `jp.co.sej.app` so the dev of the app must be informed.

**Comment by Yuki2718** [comment]
Apprently not: `https://twitter.com/zarame2000/status/1682390696794423298`

**Comment by Alex-302** [comment]
🤦🏻‍♂️

**Comment by Yuki2718** [comment]
Another `https://twitter.com/Fairy_bowwow/status/1682366011184656385`

**Comment by Yuki2718** [comment]
Here's a new case: `https://twitter.com/_trh3210/status/1698369641637380155` [This app](https://play.google.com/store/apps/details?id=jp.co.koshidaka.manekinekoapp&hl=ja&gl=US&pli=1) doesn't start if `www.google-analytics.com` is blocked on DNS, while works fine if it's blocked only on contents filter.

**Comment by Atomar0111** [comment]
Start

**Comment by Yuki2718** [comment]
Two more recent cases
https://github.com/AdguardTeam/AdguardFilters/issues/170371
https://github.com/AdguardTeam/AdguardFilters/issues/171066#issuecomment-1900359724
Though user didn't report breakage in 171066, I see total breakage (unable to launch the app) if ga is blocked on DNS.

**Comment by DandelionSprout** [comment]
Privacy lists, and by extension in this case privacy entries, tend to be broken comparatively often, so I side with OP on this one.

**Comment by Yuki2718** [comment]
A new case: https://github.com/AdguardTeam/AdguardFilters/commit/5eed11a55ce36787758f35cde029689c42b90e89

**Comment by Alain1405** [comment]
I am using AdGuard DNS filter and I cannot visit Google Analytics. Is that expected? Should the tracker be blocked?

**Comment by Alex-302** [comment]
@Alain1405 Yes. We can't block googletagmanager.com by DNS (causes many critical problems), but we can block analytics.google.com as less problematic.

But using our free [private DNS](https://adguard-dns.io/) you may unblock `analytics.google.com` and `google-analytics.com` using the query log or by adding to user rules
```adblock
@@||analytics.google.com^
@@||google-analytics.com^
```

In private DNS you may block or unblock any domain, required for your work.

<details>
<summary>Details</summary>

<img width="1294" height="751" alt="Image" src="https://github.com/user-attachments/assets/2495f0c8-a9c8-4a46-9e8c-34a9a268fd8b" />

</details>

**Comment by Yuki2718** [comment]
> But using our free [private DNS](https://adguard-dns.io/) you may unblock `analytics.google.com` and `google-analytics.com` using the query log or by adding to user rules
> 
> @@||analytics.google.com^
> @@||google-analytics.com^
> In private DNS you may block or unblock any domain, required for your work.
> 

This should be opposite, unblock by default and allow custom blocking. I can't truely recommend AG to people until this (or at least https://github.com/AdguardTeam/HostlistsRegistry/issues/609) and https://github.com/AdguardTeam/AdguardForiOS/issues/2407#issuecomment-3155398810 are solved. AG is still a tools for techie nerds and not for average user.

**Comment by milordebulldog** [comment]
Blocking analytics at DNS level really? What about everyone who has a website?

**Comment by Alex-302** [comment]
@milordebulldog you can use `Non-filtering servers`:)

**Comment by mdjr1877-droid** [comment]
#1224 #2077 #2092
